book/index.qmd

```diff
@@ -112,35 +142,43 @@ options(repos = c(
 ))
 ```
 
-Save the file then restart your R session and you're ready to go!
+Save the file, restart your R session, and you are ready to go!
 
 ```{r install verse, eval = FALSE}
 install.packages("mlr3verse")
 ```
 
-If you want latest development versions of any of our packages you can just run
+If you want latest development versions of any of our packages, run
 
 ```{r remotes, eval = FALSE}
 remotes::install_github("mlr-org/{pkg}")
 ```
 
-with `{pkg}` replaced by the package you want to install. You can see an up-to-date list of all our extension packages at [https://github.com/mlr-org/mlr3/wiki/Extension-Packages](https://github.com/mlr-org/mlr3/wiki/Extension-Packages).
+with `{pkg}` replaced with the name of the package you want to install.
+You can see an up-to-date list of all our extension packages at [https://github.com/mlr-org/mlr3/wiki/Extension-Packages](https://github.com/mlr-org/mlr3/wiki/Extension-Packages).
 
 ## Community links
 
-The mlr community is open to all and we welcome all people, from those completely new to machine learning and R up to advanced coders and data scientists. You can talk to us at anytime by joining our [Mattermost](https://lmmisld-lmu-stats-slds.srv.mwn.de/signup_email?id=6n7n67tdh7d4bnfxydqomjqspo).
+The mlr community is open to all and we welcome everybody, from those completely new to machine learning and R to advanced coders and professional data scientists.
+You can reach us on our [Mattermost](https://lmmisld-lmu-stats-slds.srv.mwn.de/signup_email?id=6n7n67tdh7d4bnfxydqomjqspo).
```

Comment:
Arguably this one is too complicated to write-out the full URL and so I'd suggest we use bitly or cutly